go.mod

```diff
@@ -1,106 +1,111 @@
 module github.com/openshift/machine-api-provider-aws
 
-go 1.18
+go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.44.105
+	github.com/aws/aws-sdk-go v1.44.177
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v1.2.3
 	github.com/golang/mock v1.6.0
-	github.com/onsi/ginkgo/v2 v2.6.0
-	github.com/onsi/gomega v1.24.0
+	github.com/onsi/ginkgo/v2 v2.6.1
```

Comment:
Can you pull in the master branch rather than a release, the master branch has a fix in it which should resolve your issue with the unit tests failing right now

Reply:
Actually `v2.7.0`, which contains your fix, is out now https://github.com/onsi/ginkgo/releases/tag/v2.7.0.
Daniel can use that.

Reply:
Yep, great, lets get them all updated to use the new release

Reply:
Update coming.

Reply:
Updated the `gingkov2` to version `2.7.0` as Dam mentioned in his comment. Tests are passing locally, looking for CI results.

Reply:
units in CI are passing now too. GG

Reply:
Great, thanks for help folks!